doc/ubuntu_release_process.md

```diff
@@ -205,12 +197,35 @@ We have daily packaging recipes that upload to the [daily ppa](https://code.laun
 
 ### When the daily recipe build fails ###
 
-The daily recipe for each release checks out master and then merges the ubuntu/$release
-and builds the package from there.  From time to time, the patches in the
-ubuntu/$release branch need to be refreshed/updated as upstream/master changes.
+The daily recipe for each release checks out tip of master, merges the ubuntu/$release
+and builds the package from there.  Daily recipe failures occur under two conditions:
+ 1. Tip of master drifts enough that patches in the ubuntu/$release branch need to be refreshed/updated/dropped as upstream/master changes.
+ 1. You just finished a [cherry-pick an upstream commit](#cherry-pick-process) into an
+    ubuntu/$release branch and need to drop that cherry pick patch so daily package
+    recipes continue to build.
```

Comment:
I don't understand this section;  If I just cherry-picked an upstream commit; daily builds aren;t broken yet; they could be if the cherry-pick patch touches files that are currently patched in the release branch.  This sort of tells me that the ``cherry-pick`` tool probably should verify if there's a collision, no?